columnflow/columnar_util.py

```diff
@@ -1389,6 +1390,9 @@ def __init__(
         if skip_func:
             self.skip_func = skip_func.__get__(self, self.__class__)
 
+        if self.check_columns_present is None:
+            self.check_columns_present = self._dependency_sets
```

Comment:
I have some producers where I might want to disable the checks. I would just need to change this to an empty set, right?
```
@producer(
    check_columns_present=set(),
)
```

Reply:
Yes, exactly. Or you could do `check_columns_present=False` instead of `set()` for cleaner semantics. (But not `None`, since that would use the default, which is to check both `uses` and `produces` columns.)